src/mpid/ch4/netmod/ofi/ofi_capability_sets.h

```diff
@@ -9,6 +9,9 @@
 #define MPIDI_OFI_OFF     0
 #define MPIDI_OFI_ON      1
 
+/* FIXME: check capability for fi_sendv */
+#define MPIDI_OFI_ENABLE_SENDV 0
```

Comment:
I think you could use `MPIDI_OFI_ENABLE_PT2PT_NOPACK` as a stand-in for support for multiple iovecs. We could even be more conservative for some capability sets (e.g., the minimal capability set has this set to `ON` at the moment).

Reply:
I am not sure about that. The semantics of `MPIDI_OFI_ENABLE_PT2PT_NOPACK` is very different from whether `fi_sendv` can be used. `NOPACK` does not always need use `sendv`.

It's interesting to do some evaluations. The only difference is in the `am_isend_short` path when the data is in the range of 1k-16k -- the gap between request header cell size and `MPIDI_OFI_DEFAULT_SHORT_SEND_SIZE`. Application usage often falls mostly in very short messages or much larger messages, so the impact of this range may be small. And even for this range, we may be able to adjust the settings to minimize the impact. I am thinking that at some point we probably should just remove `fi_sendv` usage altogether.

Reply:
Sorry, when I said "that makes sense". I meant it might make sense to just turn this off permanently for the fallback path. I'm still not a big fan of having this `#define` here.

Reply:
I see. You would like to actually make it a capability like the the rest of capabilities, right?

That's actually was my plan. I'll add the commit later.

Reply:
Either make it a capability or just remove the use of `fi_sendv` all the time.

Reply:
I see. We'll run some evaluations and then decide.